### PR TITLE
Give an example usage in the documentation

### PR DESCRIPTION
**Pull request by gz**
It took me a few minutes to figure out how to use the ZipfDistribution from reading the docs (i.e., finding the right trait implementation in the documentation).
Added an example to make it easier for someone new to get familiar with the crate.

### CODE REVIEW

**Comment by jonhoo** [comment]
Ah, that's a great idea! I'll leave some comments inline -- nothing major.

File: src/lib.rs [hunk 1]
```diff
@@ -11,6 +11,19 @@
 //! `H(N,s)` is the normalizing constant which corresponds to the generalized harmonic number
 //! of order `N` of `s`.
 //!
+//!
```
Comment (jonhoo) [inline src/lib.rs]: There's an extra empty line here that isn't necessary.

File: src/lib.rs [hunk 2]
```diff
@@ -11,6 +11,19 @@
...
+//! # Example
+//!
+//! ```
+//! extern crate rand;
```
Comment (jonhoo) [inline src/lib.rs:17]: `extern crate` shouldn't be necessary any more with the 2018 edition.

File: src/lib.rs [hunk 3]
```diff
@@ -11,6 +11,19 @@
...
+//! use rand::thread_rng;
```
Comment (jonhoo) [inline src/lib.rs]: I would leave off this `use` and call `rand::thread_rng` directly in the code above. That way it's clear that the trait is the primary thing users need to import.

**Comment by gz** [comment]
Seems like the extern crate rand is needed for doctests?
```
---- src/lib.rs -  (line 16) stdout ----
error[E0433]: failed to resolve: maybe a missing crate `rand`?
 --> src/lib.rs:17:5
  |
4 | use rand::distributions::Distribution;
  |     ^^^^ maybe a missing crate `rand`?
```

**Comment by jonhoo** [comment]
@gz looks like the tests are failing?

**Comment by jonhoo** [comment]
Ohhh, it's because I haven't upgraded the crate to the 2018 edition yet! Yup, then you need to include it :) Sorry for the unnecessary noise.

**Review by jonhoo — APPROVED** [review]

**Pull request merged**